pkg/services/ngalert/remote/forked_alertmanager.go

```diff
@@ -0,0 +1,94 @@
+package remote
+
+import (
+	"context"
+
+	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/ngalert/notifier"
+)
+
+type Mode int
+
+const (
+	ModeRemoteSecondary Mode = iota
+	ModeRemotePrimary
+)
```

Comment:
good work - I think we should probably say something on the logs when we start the forked Alertmanager. We'll need a translation layer between ints -> string but we can leave it out for now.